src/verifyproblem.py

```diff
@@ -696,7 +696,9 @@ def validate(self, testcase):
                     testcase.error('Input format validator %s crashed on input %s' % (val, testcase.infile))
                 if os.WEXITSTATUS(status) != 42:
                     testcase.error('Input format validator %s did not accept input %s, exit code: %d' % (val, testcase.infile, os.WEXITSTATUS(status)))
-
+            else:
+                self.error('Compile error for input format validators %s' % val.name)
+                self._validators.remove(val)
```

Comment:
Nitpick: `validators` should be `validator`.

A bit more substantial: I guess now one will get the "compile error" twice if running a complete check?  I would be inclined to instead just call `self.check()` before entering the for loop, to rely on the existing code.